src/App.js

```diff
@@ -467,6 +476,70 @@ function handleParentWindowMessage(event) {
   }
 }
 
+//send postMessage upon load of script
+//probably best to call this in contacts.js
+//add debugging in to figure out just how things are traveling
+//add finesse side of phonebook
+// finesse api? will have to figure out how user is created d
+function getContactsList(){
+  return contacts_list_global;
+}
+function formatNumbers(contact_list){
+  // starts with ( correct
```

Comment:
The logic here isn't right. First it ignores patterns like "5553331212" or "+15553331212", second we shouldn't assume that these are all 10 digit numbers and not, say, extension numbers for internal users.

We already have a google's phone formatting api included in the project so should use that. See `finesse_phone_api.js`, the library is `https://github.com/catamphetamine/libphonenumber-js`.

Reply:
Fixed.